docs/api_v2/api_v2_reference.apib

```diff
@@ -2492,6 +2492,9 @@ If a dataset with the given name already exists then returns its object.
 
 Returns dataset object for given dataset ID.
 
+**NOTE:** Keep in mind that attributes `itemCount` and `cleanItemCount` are not propagated right away after data were pushed into a dataset.
+There is a short period(up to 5 seconds) during which these counters may not match with exact counts in dataset items.
```

Comment:
`data were pushed` => `data are pushed`

Please add a space in `period(up to 5 seconds)` => `period (up to 5 seconds)`

Thanks!